api/v1/constructors.go

```diff
@@ -935,7 +935,7 @@ func parseCertificateInfo(spec *SystemSpec, certificates []certificates.Certific
 
 	for index, c := range certificates {
 		// Ignore OpenLDAP certificate since it's being installed during the initial unlock.
-		if c.Type == "openldap" {
+		if c.Type == "openldap" || c.Type == "ssl_ca" {
```

Comment:
We will need to install a customized ssl_ca if it is specified in the deployment-config.yaml, cannot block its installation

Reply:
I have undo the changes from constructor and updated the code of build filters